sdk/eventhub/tests.yml

```diff
@@ -7,4 +7,4 @@ extends:
     ServiceDirectory: eventhub
     SDKType: client
     TimeoutInMinutes: 190
-    Clouds: 'Public,Canary'
\ No newline at end of file
+    Clouds: 'Public,Preview,UsGov,China,Canary'
```

Comment:
If you're looking to also do this for the track one libraries, you'll need to reflect the change to `tests.data.yml` as well.